deployments/demos/zenoh-bridge/README.md

```diff
@@ -1,3 +1,37 @@
-# zenoh-bridge demo
+# Zenoh Bridge Demo
 
-See [zenoh-bridge](https://autowarefoundation.github.io/openadkit/deployments/zenoh-bridge/) deployment.
+This project demonstrates how to bridge Autoware data from Edge to Cloud using Zenoh.
+
+## Project Structure
+
+The project provides different deployment strategies to suit various testing needs:
+
+### 1. `local` (Standard Demo)
+*   Path: `./local`
+*   Description: The unified local environment supporting both Split Topology (Edge/Cloud separation) and Monolithic deployment.
+*   Usage:
+    *   Split: `./edge.sh up -d` + `./cloud.sh up -d`
+    *   Mono: `docker compose up -d`
+*   Best for: General testing, demonstrating Edge-Cloud architecture, and PR verification.
+
+### 2. `net-topo-test` (Advanced Networking)
```

Comment:
For the `net-topo-test` and `probe_tool`, we can put them here later while they are ready.